README.rst

```diff
@@ -66,6 +62,25 @@ To check if everything worked fine, you can do::
 
 and it should not give any error messages.
 
+**Parallel backends**
+
+For further instructions on installation and usage of parallel backends for using more
+than one CPU core, refer to `parallel_backends`_
+
+**Note for Windows users**
+
```

Comment:
```suggestion
We recommend using windows subsystem for linux (WSL).  For use with pure Windows, 
```

Reply:
do you have a Windows machine to give this a shot?

Reply:
Sure! Stuck in meetings today but I can give it a go later tonight for windows and linux.

Reply:
![image](https://user-images.githubusercontent.com/55253912/109564097-99503200-7aae-11eb-82ff-55c4d3f52277.png)

This showed up on the Windows install. You have to navigate to `/hnn_core/mod/` in site-packages in your conda environment and run `nrnivmodl` manually. Are they supposed to compile automatically for all systems? 

Also I'll check out mpi4py for windows, but I suspect it's going to be a power user feature...

Reply:
nopes, it should work out of the box. That's the whole point of all this pain we're taking :) For linux/mac, it works out of the box for you, right?

so [here](https://github.com/jonescompneurolab/hnn-core/blob/master/setup.py#L60) is where `nrnivmodl` is called during the `setup`. Can you verify if this works? I remember having some trouble printing or adding breakpoints in the setup file, so you might need to do some googling.

Reply:
did you try with WSL or vanilla Windows? I think if `nrnivmodl` is not found, the setup should exit with an error ... not sure how we can make that happen

Reply:
This is on vanilla Windows 10. I think WSL works fine  but I need to double check. Anyways here's the problem:
![image](https://user-images.githubusercontent.com/55253912/109581876-1c7f8100-7acb-11eb-931f-86fbc840d264.png)

Looks like Anaconda creates different directory structure on windows. We can add an if clause to create two different `mod_path` variables, however `hnn_core.__file__` returns a path with a lowercase 'l' for /lib/, whereas on Windows the path uses an uppercase 'L'.

Is `hnn_core.__file__` semi hardcoded? Or do we need a special workaround to preserve case-sensitive file paths.

Reply:
weird ... quick googling seems to suggest that windows does not distinguish between upper and lower case. Can you give it a quick shot? If it works, you can make a pull request to my `release_vp1` branch. Otherwise, we might have to postpone Windows support for later

Reply:
Will do. Seems like it should just be a matter of removing the extra `/hnn_core/` from the path, but I'm not gambling on those odds...

Reply:
Ok so it seemed to work when I ran it in an external script. Is it possible to re-upload to test.pypi to see if it goes smoother?

Also here's the actual error that I believe is thrown during the pip install:
![image](https://user-images.githubusercontent.com/55253912/109586252-dfb78800-7ad2-11eb-8649-60881a0a458e.png)

Reply:
Yep I can upload again to `test.pypi` ... can you send a PR to this branch?

Reply:
Oh sorry I just pushed a commit to the branch, should I revert it?

Reply:
oh indeed, sorry I had forgotten you could push. It's all good. Uploaded to test.pypi again. Give it a shot

Reply:
Hmmm that didn't seem to cut it. Perhaps there's a lag for test.pypi to update? Not sure how to check that since setup.py is not included in the pip install.

I'll try again in the morning, also I need to read through this but this may explain the issue:
https://docs.python.org/3/install/#how-installation-works

Reply:
Just download the tar from pypi directly and you should have the setup file there: https://test.pypi.org/project/hnn-core/#files

Reply:
also if you tell me your test.pypi username I can add you there so you can try uploading yourself